azure-pipelines.yml

```diff
@@ -82,9 +64,118 @@ jobs:
       command: test
       projects: '$(folder)/**/tests/*.csproj'
       arguments: '--configuration Release'
-- job: package
-  displayName: Package
+
+#
+# Run compliance tooling
+#
+- job: sdl
+  displayName: SDL Tasks
+  pool:
+    name: Hosted Windows 2019 with VS2019
+    demands:
+    - msbuild
+    - npm
+  steps:
+  - task: DotNetCoreCLI@2
+    displayName: Restore
+    inputs:
+      command: restore
+      projects: 'Industrial-IoT.sln'
+  - task: MSBuild@1
+    displayName: Build
+    inputs:
+      solution: 'Industrial-IoT.sln'
+      configuration: Release
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
+    displayName: 'Credential Scanner'
+    inputs:
+      debugMode: false
+    continueOnError: true
+  - task: ComponentGovernanceComponentDetection@0
+    displayName: 'Component Detection'
+  - task: 'msospo.ospo-extension.8d7f9abb-6896-461d-9e25-4f74ed65ddb2.notice@0'
+    displayName: Generate 3rd Party Notices
+    inputs:
+      outputfile: thirdpartynotices.txt
+    continueOnError: true
+  - task: TSLint@1
+    displayName: 'Run TSLint'
+    inputs:
+      Files: '**\*.ts;**\*.js'
+      ExcludeFiles: 'api\**'
+    continueOnError: true
+  - task: BinSkim@3
+    displayName: 'BinSkim '
+    inputs:
+      InputType: Basic
+      AnalyzeTarget: '*.IIoT.*.dll;*.exe'
+    continueOnError: true
+  - task: AutoApplicability@1
+    displayName: 'Auto Applicability'
+    inputs:
+      ExternalRelease: true
+      InternalRelease: true
+      IsService: true
+    continueOnError: true
+  - task: VulnerabilityAssessment@0
+    displayName: 'Run Vulnerability Assessment'
+    continueOnError: true
+    condition: succeededOrFailed()
+  - task: PoliCheck@1
+    displayName: PoliCheck
+    inputs:
+      targetType: F
+    continueOnError: true
+  - task: RoslynAnalyzers@2
+    displayName: 'Run Roslyn Analyzers'
+    continueOnError: true
+  - task: SdtReport@1
+    displayName: 'Create Security Analysis Report'
+    inputs:
+      BinSkim: true
+      CredScan: true
+      PoliCheck: true
+      RoslynAnalyzers: true
+    continueOnError: true
+    condition: succeededOrFailed()
+  - task: PostAnalysis@1
+    displayName: 'Post Analysis'
+    inputs:
+      BinSkim: true
+      CredScan: true
+      RoslynAnalyzers: true
+  - script: |
+      if not "%BUILD_SOURCEBRANCHNAME%" == "master" echo ##vso[task.setvariable variable=MY_BRANCH;]develop
```

Comment:
Do we still want to make use of develop branch?

Reply:
Everything != master == develop, this is just the name of the TSA stamp, and we have 2.